lib/tasks/publishing_api.rake

```diff
@@ -32,6 +32,56 @@ namespace :publishing_api do
       ))
     end
   end
+
+  desc "Send publishable item links to Publishing API"
+  task publishing_api_patch_links: :environment do
+    $stdout.puts "# Sending all items' links to Publishing API"
+    $stdout.puts "# Ruby code is output for failed operations"
+    $stdout.puts
+
+    def patch_links(item)
+      content_id = item.content_id
+      links = PublishingApiPresenters.presenter_for(item).links
+      if links
+        Whitehall.publishing_api_v2_client.patch_links(content_id, {links: links})
```

Comment:
Panopticon also does this:

https://github.com/alphagov/panopticon/blob/7c00f043720f354d59e15519006771b6fdaf8a5a/lib/tasks/router.rake

Tagging migration verifier uses a gem that provides exponential backoff:

https://github.com/alphagov/tagging-migration-verifier/blob/1c5b858dc639d9f79136aedc48f5de2253f8ca93/lib/download.rb#L21-L24